### DISCUSSION TITLE
Where is the output of `detect predict`?

### DISCUSSION THREAD

**Discussion by vromero** [ultralytics/ultralytics#935] [Q&A] [answered]
Deal all,

I'm used to get a result in `runs` when I run predict in v7 but here I'm running 
```
yolo detect predict model=runs\detect\train4\weights\best.pt source=images\val\fff90a48-c03f-400a-8cdb-e49a0aeafb3d.png device=cpu
Ultralytics YOLOv8.0.34  Python-3.10.9 torch-1.12.1 CPU
Model summary (fused): 168 layers, 3005843 parameters, 0 gradients, 8.1 GFLOPs

image 1/1 C:\Users\victo\datasets\windows11\images\val\fff90a48-c03f-400a-8cdb-e49a0aeafb3d.png: 512x512 1 window, 82.0ms
Speed: 1.0ms pre-process, 82.0ms inference, 1.0ms postprocess per image at shape (1, 3, 512, 512)
```

(Tried both with GPU and CPU) and I see no result in runs nor in the console.

Thanks!

**Comment by AyushExel** [comment, marked as answer]
Pass show=True

**Reply by vromero** [reply]
Thanks @AyushExel, 

A window with the result appeared and then dissapeared inmmediatelly afterwards. 
Is there a comprehensive list of parameters? Can't find it.

Thanks

**Reply by AyushExel** [reply]
pass save=True to save the result predictions. 
All parameters can be seen here in the official docs - https://docs.ultralytics.com/cfg/

**Reply by vromero** [reply]
Thanks @AyushExel, this did the trick.

**Reply by AyushExel** [reply]
https://docs.ultralytics.com/cfg/
